src/app/login/login.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 d-flex align-items-center justify-content-center">
+
+      <form [formGroup]="logForm" (ngSubmit)="onSubmit()">
+
+        <div class="text-center">
+          <h2>LOGIN</h2>
+          <p class="text-secondary">Please enter your credentials</p>
```

Comment:
I recommend moving message content/all labels to a file, we normally use this library: https://www.npmjs.com/package/@ngx-translate/core